Automattic/vip-go-mu-plugins
# Search: Do not sync if index does not exist

## Description
In the use case described in https://github.com/Automattic/vip-go-mu-plugins/pull/2687#issuecomment-988566800, we sync even if the index does not exist. This filter will kill syncing actions if the index does not exist for the indexable.

## Changelog Description

### Plugin Updated: Enterprise Search

Use `ep_sync_indexable_kill` filter to check if index exists before attempting to sync.


## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test
1) Using a multisite ES (not in network mode), create a new subsite
2) Verify that the index does not exist yet for it via `wp vip-search get-indexes`
3) Create a post
4) Repeat step 2 to ensure that an index does not get created
